### PR TITLE
tint color

### PR DESCRIPTION
**Pull request by 0o0xia**

### CODE REVIEW

File: Phanpy/Features/Settings/Setting.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//
+//  Setting.swift
+//  Phanpy
+//
+//  Created by 邵景添 on 2018/12/28.
+//
+
+import UIKit
+
+protocol Settingable {
```
Comment (kylinroc) [inline Phanpy/Features/Settings/Setting.swift]: Settingable...

I think what you mean is `SettingItem` or something?

**Comment by 0o0xia** [comment]
thinking...

File: Phanpy/Features/Settings/SettingsManager.swift [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    case red
+    case orange
+    case yellow
+    case green
+    case tealBlue
+    case blue
+    case pink
+}
+
+final class SettingsManager {
```
Comment (kylinroc) [inline Phanpy/Features/Settings/SettingsManager.swift]: Avoid use `*Manager` in name if possible.

 I think the proper name for this type should be `Settings` (or `UserSettings`, `AppSettings` etc), it is clear and straightforward.

File: Phanpy/Features/Settings/SettingCell.swift [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        textLabel?.text = setting.title
+        imageView?.image = setting.icon
+    }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        imageView?.bounds.size = CGSize(width: 29, height: 29)
+        imageView?.center.y = contentView.center.y
+        imageView?.layer.cornerRadius = 6
+        imageView?.layer.masksToBounds = true
```
Comment (kylinroc) [inline Phanpy/Features/Settings/SettingCell.swift]: Do not mess around with those `UITableViewCell`'s properties

File: Phanpy/Features/Settings/SettingsManager.swift [hunk 4]
```diff
@@ -0,0 +1,86 @@
+//
+//  SettingsManager.swift
+//  Phanpy
+//
+//  Created by 邵景添 on 2018/12/28.
+//
+
+import UIKit
+
+enum ThemeColorType: String {
```
Comment (kylinroc) [inline Phanpy/Features/Settings/SettingsManager.swift]: Why we have `ThemeColor` and `ThemeColorType`? It is confusing.

File: Phanpy/Features/Settings/SettingsManager.swift (hunk 2 above)
Comment (0o0xia) [reply Phanpy/Features/Settings/SettingsManager.swift]: good...

File: Phanpy/Features/Settings/SettingsManager.swift (hunk 4 above)
Comment (0o0xia) [reply Phanpy/Features/Settings/SettingsManager.swift]: delete ~

**Pull request merged**